Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -355,14 +355,16 @@ class Connection : public ThreadSafeRefCounted<Connection, WTF::DestructionThrea
     void dispatchMessageReceiverMessage(MessageReceiver&, std::unique_ptr<Decoder>&&);
     // Can be called from any thread.
     void dispatchDidReceiveInvalidMessage(MessageName);
+
+    size_t getPendingMessageCountForTesting();
```

Comment:
Per WebKit coding style, we don't use the "get" prefix for getters in WebKit. This should be named `pendingMessageCountForTesting()`.

Reply:
Also, can this be const? Seems odd for a getter to not be const. We may have to mark the Lock as mutable.